### DISCUSSION TITLE
Not able to code space

### DISCUSSION THREAD

**Discussion by SURYAROUSHAN** [community/community#67297] [Codespaces]
### Select Topic Area

Question

### Body

Create New Codespace: List devcontainer definitions request failed with status 400. Correlation ID: FFEB:14C9:231FD2D:4711020:64F9CD2F

**Comment by vipulkumarstm** [comment]
Refresh it again

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
